nef/Extensions/AppDelegate+Notifications.swift

```diff
@@ -62,12 +62,14 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
     }
     
     func processNotification(_ userInfo: [String: Any], action: String) -> EnvIO<NotificationConfig, NefNotification.Error, NefNotification.Response> {
-        guard let image = userInfo[NefNotification.UserInfoKey.imageData] as? Data else { return EnvIO.raiseError(.noImageData)^ }
+        guard let image = userInfo[NefNotification.UserInfoKey.imageData] as? Data else {
+            return EnvIO.raiseError(.noImageData)^
+        }
         
         switch action {
         case NefNotification.Action.saveImage.identifier:
             return image
-                .persist(command: .clipboardCarbon(code: ""))
+                .persist(command: .exportSnippetToClipboard(selection: ""))
```

Comment:
This seems incorrect, as the value has to be created with an empty String to be able to invoke this method. I guess you are looking for the corresponding `MenuEditorCommand` as you only need the key.

Reply:
I am only using the key, right now, but in this point, the action does not come from any menu editor so it should send the code selection too. I am going to update it to send the code selection.

Reply:
But it is out of this PR, it will be done in the [next one](https://github.com/bow-swift/nef-plugin/pull/18)